src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -204,6 +204,8 @@ public function toolbarAction(Request $request, $token)
             $url = $this->generator->generate('_profiler', array('token' => $token));
         } catch (\Exception $e) {
             // the profiler is not enabled
+        } catch (\Throwable $e) {
+            // the profiler is not enabled
```

Comment:
There are 3 exceptions in the `UrlGeneratorInterface`. IMHO it would be better to catch those three and ignore the rest?